lib/PHPCfg/AstVisitor/NameResolver.php

```diff
@@ -42,10 +42,19 @@ class NameResolver extends NameResolverParent
         'resource',
         'callable',
     ];
+    
+    protected $anonymousClasses = 0;
 
     public function enterNode(Node $node)
-    {
+    {        
         parent::enterNode($node);
+
+        if (property_exists($node, "name") && is_null($node->name)) {
```

Comment:
Please do something like `instanceof Class_` here instead of checking property_exists.